defmt/src/lib.rs

```diff
@@ -361,6 +361,31 @@ pub use defmt_macros::timestamp;
 ///         const ABC = Self::A.bits | Self::B.bits | Self::C.bits;
 ///     }
 /// }
+///
+/// defmt::info!("Flags::ABC: {}", Flags::ABC);
+/// defmt::info!("Flags::empty(): {}", Flags::empty());
+/// ```
+pub use defmt_macros::bitflags;
+
+/// Generates a bitflagsv2 structure that can be formatted with defmt. Users of the defmt crate can 
```

Comment:
This makes it sound like `bitflagsv2` is a different crate to `bitflags`, which is not the case, it's the same `bitflags` crate, but just using version 2.X.X.
Please try to formulate this clearer.